search-api/src/search_api/services/authz.py

```diff
@@ -36,6 +37,8 @@
 BCOL_HELP = 'helpdesk'
 SBC_STAFF = 'sbc_staff'
 
+auth_cache = Cache(config={'CACHE_TYPE': 'SimpleCache'})
```

Comment:
Can you update the config file with cache type and cache dir same as BOR and initialize it the same way here? 
We don't want to use 'SimpleCache' because it isn't thread safe so it won't work as expected in the deployment where it runs with multiple processors/threads. That's why we use the 'FileSystemCache' in BOR. I also made this mistake when implementing flask caching previously

Reply:
And it's good to have it set in the config so that we can update it as needed without needing to make code changes